tools/mo/requirements_tf2.txt

```diff
@@ -1,7 +1,8 @@
 numpy>=1.16.6,<1.25.0
-tensorflow>=2.5,<=2.11.0
+tensorflow>1.15.5,<=2.13.0rc0
```

Comment:
let us not use RC in our requirements files. They are in pre-production state.

Reply:
It's a draft. I used rc0 in order to debug the potential issues, because if we wait for new TensorFlow to officially release we may not have enough time to finish the rest of the work. That way I can continue working, using rc0 as a temporary workaround. 

The version will be switched to official release before marking the PR ready for review and before merge.

Reply:
The current decision is to switch off all pdpd tests on master until new version is released. See 95904.

Reply:
to resolve a discussion - TF 2.12 which satisfies our requirements has been released and this notation is not needed anymore and will be replaced with released-based one.